code/game/objects/items/legcuffs.dm

```diff
@@ -92,3 +93,45 @@
 					var/mob/living/simple_animal/SA = AM
 					SA.health -= 20
 	..()
+
+
+/obj/item/restraint/legcuffs/xeno_restraints
+	name = "xeno restraints"
+	desc = "Use this to hold xenomorphic creatures moderately-safely."
+	flags_equip_slot = SLOT_WAIST
+	throwforce = 5
+	w_class = SIZE_SMALL
+	throw_range = 5
+	matter = list("metal" = 500)
+
+	breakouttime = 2 MINUTES
+
+/obj/item/restraint/legcuffs/xeno_restraints/apply_legcuffs(mob/living/carbon/target, mob/user)
+	return FALSE
+
+/obj/item/restraint/legcuffs/xeno_restraints/attack(mob/living/carbon/target_mob, mob/user as mob)
+	if(!istype(target_mob, /mob/living/carbon/xenomorph))
+		to_chat(user, SPAN_DANGER("The cuffs do not fit!"))
+		return
+	if(!target_mob.legcuffed)
+		var/turf/user_loc = user.loc
```

Comment:
user.loc is not guaranteed to be a turf, if you want a turf specifically use get_turf(user)